travis/apis.py

```diff
@@ -0,0 +1,180 @@
+# coding: utf-8
+
+import base64
+import os
+import tempfile
+import time
+import json
+from contextlib import contextmanager
+import requests
+
+
+class ApiException(Exception):
+    pass
+
+
+class Request(object):
+
+    def __init__(self):
+        self.session = requests.Session()
+
+    def _check(self):
+        if not self._token:
+            raise ApiException("WARNING! Token not defined exiting early.")
+        self.session.headers.update({
+            'Accept': 'application/json',
+            'User-Agent': 'mqt',
+            'Authorization': 'Token %s' % self._token
+        })
+        self._request(self.host)
+
+    def _request(self, url, payload=None, is_json=True, patch=False):
+        try:
+            if not payload and not patch:
+                response = self.session.get(url)
+            elif patch:
+                response = self.session.patch(url, data=payload)
+            else:
+                response = self.session.post(url, data=payload)
+            response.raise_for_status()
+        except requests.RequestException as error:
+            raise ApiException(str(error))
+        return response.json() if is_json else response
+
+
+class WeblateApi(Request):
+
+    def __init__(self):
+        super(WeblateApi, self).__init__()
+        self.repo_slug = None
+        self.branch = None
+        self._token = os.environ.get("WEBLATE_TOKEN")
+        self.host = os.environ.get(
+            "WEBLATE_HOST", "https://weblate.vauxoo.com/api")
```

Comment:
@dreispt 
Good point, You are right
I replace vauxoo.com for odoo-community.org